iphone/cli/commands/_build.js

```diff
@@ -2588,6 +2588,11 @@ iOSBuilder.prototype.checkIfNeedToRecompile = function checkIfNeedToRecompile()
 			return true;
 		}
 
+		if (this.target === 'dist-adhoc' || this.target === 'dist-appstore') {
+			this.logger.info(__('Forcing rebuild: distribution builds require \'xcodebuild\' to be run so that resources are copied into the archive'));
```

Comment:
No other "Forcing rebuild:" message has the following sentence capitalized. It may be consistently wrong, but at least it's consistent.